hw2/utils.ts

```diff
@@ -0,0 +1,72 @@
+import { User } from './models/user.model';
```

Comment:
It's a way too common name of a file.
`utils.ts` -> `user.store.ts` cause it's about the storage of the user data.